Content.Shared/GameObjects/Components/Mobs/State/SharedAliveMobState.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Content.Shared.GameObjects.Components.Mobs.State
+{
+    /// <summary>
+    ///     The standard state an entity is in; no negative effects.
+    /// </summary>
+    public abstract class SharedAliveMobState : BaseMobState
```

Comment:
Maybe change the name to something like SharedNormalMobState since crit is technically alive

Reply:
Done.